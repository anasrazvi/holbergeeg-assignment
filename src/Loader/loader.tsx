import React from 'react';
import Loader from './LoadSpinner.module.css';

export const LoadSpinner = () => (
  <div className={Loader.loader}></div>
);

export default LoadSpinner;