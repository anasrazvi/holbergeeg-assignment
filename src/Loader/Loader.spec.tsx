import renderer from 'react-test-renderer';
import {LoadSpinner} from '../Loader/loader';

describe("Loader Spinner", () => {
    test('renders loader', () => {
      const loaderSpinner = renderer.create(<LoadSpinner></LoadSpinner>).toJSON();
      expect(loaderSpinner).toMatchSnapshot();
    });
});