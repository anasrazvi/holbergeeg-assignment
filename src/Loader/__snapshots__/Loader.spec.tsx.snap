// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader Spinner renders loader 1`] = `
<div
  className="loader"
/>
`;
